apps/site/next-data/generators/__tests__/blogData.test.mjs

```diff
@@ -0,0 +1,189 @@
+import { Readable } from 'node:stream';
+
+import generateBlogData from '@/next-data/generators/blogData.mjs';
+
+let files = [];
+
+jest.mock('node:path', () => ({
+  ...jest.requireActual('node:path'),
+  join: jest.fn((_base, filePath) => filePath),
+}));
```

Comment:
I don't think we need to mock path

Reply:
I was mocking `join` so that I could ignore the base path here: https://github.com/nodejs/nodejs.org/blob/eacdf5a0593d5abf585fc9fce9adeb7c01715b0b/apps/site/next-data/generators/blogData.mjs#L65 in the `createReadStream` mock

it is a bit dirty actually... :confused: 

I've removed the mocking in favor of resolving the path in the `createReadStream` mock instead :slightly_smiling_face: 

https://github.com/nodejs/nodejs.org/pull/7618/commits/fec585fdf8a45d6b0f87a1094f76a9c209ee884d